tests/system/test_observability_options.py

```diff
@@ -241,30 +272,70 @@ def select_in_txn(txn):
 
             got_events.append((event.name, evt_attributes))
 
+    return got_statuses, got_events
+
+
+@pytest.mark.skipif(
+    not _helpers.USE_EMULATOR,
+    reason="Emulator needed to run this test",
+)
+@pytest.mark.skipif(
+    not HAS_OTEL_INSTALLED,
+    reason="Tracing requires OpenTelemetry",
+)
+def test_database_partitioned():
+    from opentelemetry.trace.status import StatusCode
+
+    db, trace_exporter = create_db_trace_exporter()
+
+    try:
+        db.execute_partitioned_dml("UPDATE NonExistent SET name = 'foo' WHERE id > 1")
+    except Exception:
+        pass
+
+    got_statuses, got_events = finished_spans_statuses(trace_exporter)
     # Check for the series of events
     want_events = [
         ("Acquiring session", {"kind": "BurstyPool"}),
         ("Waiting for a session to become available", {"kind": "BurstyPool"}),
         ("No sessions available in pool. Creating session", {"kind": "BurstyPool"}),
         ("Creating Session", {}),
+        ("Starting BeginTransaction", {}),
         (
-            "Transaction was aborted in user operation, retrying",
-            {"delay_seconds": "EPHEMERAL", "cause": "EPHEMERAL", "attempt": 1},
+            "exception",
+            {
+                "exception.type": "google.api_core.exceptions.InvalidArgument",
+                "exception.message": "400 Table not found: NonExistent [at 1:8]\nUPDATE NonExistent SET name = 'foo' WHERE id > 1\n       ^",
+                "exception.stacktrace": "EPHEMERAL",
+                "exception.escaped": "False",
+            },
+        ),
+        (
+            "exception",
+            {
+                "exception.type": "google.api_core.exceptions.InvalidArgument",
+                "exception.message": "400 Table not found: NonExistent [at 1:8]\nUPDATE NonExistent SET name = 'foo' WHERE id > 1\n       ^",
+                "exception.stacktrace": "EPHEMERAL",
+                "exception.escaped": "False",
+            },
```

Comment:
May be I missed some context,
Can you help me understand why we see 2 exceptions here instead of only one? This was fixed here right?
https://github.com/googleapis/python-spanner/pull/1207/files#r1862942500

Reply:
An exception was thrown from the lower levels and with the chaining across with calls, it still gets propagated. The case we fixed was entirely different because in https://github.com/googleapis/python-spanner/issues/1246 that status was being overwritten back to OK.